projects/1.18/assets/irongenerators/lang/zh_cn.json

```
@@ -0,0 +1,57 @@
+{
+  "item.moregenerators.netherstar_upgrade": "网络升级",
+  "block.moregenerators.iron_generator": "铁制发电机",
+  "block.irongenerators.refined_obsidian_generator": "强化黑曜石发电机",
+  "block.moregenerators.certus_quartz_generator": "赛特斯石英水晶发电机",
+  "item.moregenerators.emerald_upgrade": "Emerald升级",
+  "block.moregenerators.icon": "Icon",
```

```suggestion
  "block.moregenerators.icon": "模组标志",
```
大概是创造物品栏的头图，就看着整一个（）